spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/SparkWriteOptions.java

```diff
@@ -77,4 +77,7 @@ private SparkWriteOptions() {}
 
   // Isolation Level for DataFrame calls. Currently supported by overwritePartitions
   public static final String ISOLATION_LEVEL = "isolation-level";
+
+  // Checks if case of schema needs to be considered during
```

Comment:
doc is incomplete?

Reply:
I think here it only applies during "mergeSchema or if allowAnySchema" is in use. I'm also not sure if we want to expose this here, User's should just be using Spark's case-sensitive property. I don't think we want them changing it for individual operations from the session default. 

I think we are probably better off just dropping this as an option